checkov/terraform/checks/resource/aws/S3Encryption.py

```diff
@@ -19,12 +19,14 @@ def scan_resource_conf(self, conf):
         """
         if 'server_side_encryption_configuration' in conf.keys():
             sse_block = conf['server_side_encryption_configuration']
-            if 'rule' in sse_block[0].keys():
-                rule_block = sse_block[0]['rule']
-                if 'apply_server_side_encryption_by_default' in rule_block[0].keys():
-                    encryption_block = rule_block[0]['apply_server_side_encryption_by_default']
-                    if  'sse_algorithm' in encryption_block[0].keys():
-                        return CheckResult.PASSED
+            if type(sse_block[0]) is dict:
```

Comment:
@arielkru I believe that if it isn't a `dict`, we won't be able to assess it - so we should let it pass, not fail. WDYT?